example/app.js

```diff
@@ -2,16 +2,22 @@ import React from 'react';
 import ReactDom from 'react-dom';
 import Tweet from '../src/components/Tweet/Tweet.js';
 import './_app.css';
-import tweets from './tweets'
+import tweets from './tweets';
 
 const linkProps = { target: '_blank' };
 
-ReactDom.render((
-  <div className="ExamplePage" style={{'width': '590px', 'margin': '0 auto'}}>
-    <div className="tweet-stream" style={{'width': '100%'}}>
-      {tweets.map((t, i) => (
-        <Tweet autoPlay={true} data={t} key={i} linkProps={linkProps} />
-      ))}
+const filteredTweets = tweets.filter(tweet => {
```

Comment:
I filtered the tweets out here to ease the testing of the styled component migration of Video.js.